src/argus_htmx/static/styles.css

```diff
@@ -1,3677 +0,0 @@
-/*
-! tailwindcss v3.4.10 | MIT License | https://tailwindcss.com
-*/
-
-/*
```

Comment:
Don't forget to squash your commits (eg when merging), otherwise this big file remains in the git history